source/run-command.txt

```diff
@@ -0,0 +1,197 @@
+        "localTime" : ..., 
+        "logicalSessionTimeoutMinutes" : ..., 
+        "connectionId" : ..., 
+        "minWireVersion" : ..., 
+        "maxWireVersion" : ..., 
+        "readOnly" : ..., 
+        "ok" : ..., 
+        "$clusterTime" : ..., 
+        "signature" : ... 
+       } 
+
+.. cpp-command-options:
+
+Command Options
+---------------
+
+To customize command execution behavior, you can set options in the command 
+document that you pass to the ``run_command()`` method. To learn more about 
+a command and the options that it accepts, locate the command and follow the 
+corresponding link on the :manual:`Database Commands </reference/command/>` page 
+in the {+mdb-server+} manual. 
+
+For example, you can instruct the ``connectionStatus`` command 
+to return the full set of privileges that currently-authenticated users possess by setting the ``showPrivileges`` 
+option to ``true`` in the command document: 
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/run-command.cpp
+      :start-after: start-run-connectionStatus
+      :end-before: end-run-connectionStatus
+      :language: cpp
+      :dedent:
+   
+   .. output:: 
+      :language: cli
+      :visible: false
+
+      { 
+        "authInfo" : { "authenticatedUsers" : [ { "user" : ..., "db" : ... } ], 
+        "authenticatedUserRoles" : [ { "role" : ..., "db" : ... } ], 
+        "authenticatedUserPrivileges" : [ 
+                                          { "resource" : { "db" : "", "collection" : "" }, "actions" : [ ... ] }, 
+                                          { "resource" : { "db" : "config", "collection" : "system.sessions" }, "actions" : [ ... ] }, 
+                                          ..., 
+                                          { "resource" : { "db" : "", "collection" : "" }, "actions" : [ ... ] }
+        ] 
+      }, 
+         "ok" : 1 
+      }
+
+Response
+--------
+
+The ``run_command()`` method returns a ``bsoncxx::document::value`` object that contains
+the response from the database after the command has been executed. Each
+database command performs a different function, so the response content
+can vary across commands. However, every response contains documents
+with the following fields:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 70
+
+   * - Field
+     - Description
+
+   * - <command result>
+     - Provides fields specific to the database command. For example,
+       ``count`` returns the ``n`` field and ``explain`` returns the
+       ``queryPlanner`` field.
+
+   * - ``ok``
+     - Indicates whether the command has succeeded (``1``)
+       or failed (``0``).
+
+   * - ``operationTime``
+     - Indicates the logical time of the operation. MongoDB uses the
+       logical time to order operations.
+       
+       To learn more about logical time, see our :website:`blog post about
+       the Global Logical Clock </blog/post/transactions-background-part-4-the-global-logical-clock>`.
```

Comment:
This link goes to the transactions page of the Server manual instead of a blog post. I would suggest rewording this if the transactions page is what you're trying to link to, or correct the link to point to the blog post

Reply:
Ah, it looks like the original blog post is either no longer up or no longer accessible. (A post called **Transactions Background Part 4: The Global Logical Clock**). I removed this link from the page.